### ISSUE TITLE
Recurring Task #113659 for MMOCore generating exception ERROR

### ISSUE THREAD

**Issue by Brickwrath** [mmopluginteam/mmocore#102]
This console error message runs once a second starting at the completion of server boot ("Done") until shutdown.
I believe I'm running latest MMOLib, Core, MythicMobs, and Items as well as latest Tuinity paperclip build.

`[19:22:32 WARN]: [MMOCore] Task #113659 for MMOCore v1.4.2-SNAPSHOT generated an exception
java.lang.IllegalArgumentException: Player must be online
        at org.apache.commons.lang.Validate.notNull(Validate.java:192) ~[patched_1.15.2.jar:git-Tuinity-"3cdf7e4"]
        at net.mmogroup.mmolib.api.player.MMOPlayerData.getPlayer(MMOPlayerData.java:70) ~[?:?]
        at net.Indyuce.mmocore.api.player.PlayerData.getPlayer(PlayerData.java:166) ~[?:?]
        at net.Indyuce.mmocore.api.event.PlayerDataEvent.<init>(PlayerDataEvent.java:11) ~[?:?]
        at net.Indyuce.mmocore.api.event.PlayerCombatEvent.<init>(PlayerCombatEvent.java:13) ~[?:?]
        at net.Indyuce.mmocore.api.player.CombatRunnable.run(CombatRunnable.java:29) ~[?:?]
        at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.15.2.jar:git-Tuinity-"3cdf7e4"]
        at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.15.2.jar:git-Tuinity-"3cdf7e4"]
        at net.minecraft.server.v1_15_R1.MinecraftServer.b(MinecraftServer.java:1324) ~[patched_1.15.2.jar:git-Tuinity-"3cdf7e4"]
        at net.minecraft.server.v1_15_R1.DedicatedServer.b(DedicatedServer.java:432) ~[patched_1.15.2.jar:git-Tuinity-"3cdf7e4"]
        at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:1239) ~[patched_1.15.2.jar:git-Tuinity-"3cdf7e4"]
        at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:970) ~[patched_1.15.2.jar:git-Tuinity-"3cdf7e4"]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_261]
`

Server:
 - Running Tuinity latest build
 - Running latest releases of all MM-related elements

**Comment by kelvin-junilson** [comment]
Happening on paper 1.15.2 latest build as well

**Comment by blek-id** [comment]
Happens on 1.15.2 paper build as well

**Comment by mibby** [comment]
Ah, I was going to report something similar.

MMOLib 1.2.5 Build 2
MMOCore-1.4.2-SNAPSHOT dev 218
MMOItems-5.5.4 dev 299
Tuinity dev 316 (Paper 1.16.1) (https://github.com/Spottedleaf/Tuinity/actions)

Error occurred on shutdown.
```
[09:52:41] [Server thread/ERROR]: Error occurred while disabling MMOCore v1.4.2-SNAPSHOT (Is it up to date?)
java.lang.IllegalArgumentException: Player must be online
	at org.apache.commons.lang.Validate.notNull(Validate.java:192) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at net.mmogroup.mmolib.api.player.MMOPlayerData.getPlayer(MMOPlayerData.java:70) ~[?:?]
	at net.Indyuce.mmocore.api.player.PlayerData.getPlayer(PlayerData.java:166) ~[?:?]
	at net.Indyuce.mmocore.api.quest.PlayerQuests.resetBossBar(PlayerQuests.java:188) ~[?:?]
	at net.Indyuce.mmocore.MMOCore.onDisable(MMOCore.java:364) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:265) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:415) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:516) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at org.bukkit.plugin.SimplePluginManager.disablePlugins(SimplePluginManager.java:502) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at org.bukkit.plugin.SimplePluginManager.disablePlugins(SimplePluginManager.java:495) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at org.bukkit.craftbukkit.v1_16_R1.CraftServer.disablePlugins(CraftServer.java:434) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at net.minecraft.server.v1_16_R1.MinecraftServer.stop(MinecraftServer.java:774) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at net.minecraft.server.v1_16_R1.DedicatedServer.stop(DedicatedServer.java:700) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1047) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Tuinity-"d3dc35c"]
	at java.lang.Thread.run(Thread.java:832) [?:?]
```

**Comment by jacobmepstein** [comment]
Same as #110. Should be fixed in MMOCore 1.4.3 and MMOItems 5.5.4 build 3370.
